typescript/sdk/src/token/EvmERC20WarpModule.ts

```diff
@@ -182,12 +246,24 @@ export class EvmERC20WarpModule extends HyperlaneModule<
     const deployer = new HypERC20Deployer(multiProvider);
     const deployedContracts = await deployer.deployContracts(chainName, config);
 
-    return new EvmERC20WarpModule(multiProvider, {
+    const warpModule = new EvmERC20WarpModule(multiProvider, {
       addresses: {
         deployedTokenRoute: deployedContracts[config.type].address,
       },
       chain,
       config,
     });
+
+    // Enroll Remote Routers
+    if (config.remoteRouters && !isObjEmpty(config.remoteRouters)) {
+      const enrollRemoteTx = await warpModule.updateRemoteRouters(
+        await warpModule.read(),
+        config,
+      );
+
+      await multiProvider.sendTransaction(chain, enrollRemoteTx[0]); // updateRemoteRouters is always a single tx
```

Comment:
and if we want to get fancy to remove the comment:
```suggestion
      const onlyTxIndex = 0;
      await multiProvider.sendTransaction(chain, enrollRemoteTx[onlyTxIndex]);
```

Reply:
updated.